### ISSUE TITLE
IC2 missing - stacktrace output to log

### ISSUE THREAD

**Issue by RAnders00** [OpenMods/OpenPeripheral#237]
http://pastie.org/private/tywujecyoi0c3zxxxonryw

This is output on game load when IC2 is not installed.
It would be nicer if you could just check whether IC2 is loaded (by utilizing a FML call) and output a nice INFO call that the IC2 plugin will be disabled.

**Comment by boq** [comment]
We already have such code - see [here](https://github.com/OpenMods/OpenPeripheral-Integration/blob/master/src/main/java/openperipheral/integration/ApiIntegrationModule.java#L18) and [here](https://github.com/OpenMods/OpenPeripheral-Integration/blob/master/src/main/java/openperipheral/integration/ic2/ModuleIC2Api.java#L12) (BTW, as you can probably see, it's OpenPeripheral-Integration problem, not OpenPeripheral-Core).

But this problem is caused by other mods (confirmed at least Railcraft, see CovertJaguar/Railcraft#579) that not only contain API inside (bad) but also partial one (terrible) - this causes API to be registered by FML (and visible to OpenPeripheral-Integration), but broken, since classes are missing.

tl;dr; We already did you proposed, but other mods break it.
